packages/view-types/statistical-plots/src/ExpressionHistogramSubscriber.js

```diff
@@ -37,13 +39,27 @@ export function ExpressionHistogramSubscriber(props) {
     featureType,
     featureValueType,
     featureSelection: geneSelection,
+    additionalObsSets: additionalCellSets,
+    obsSetColor: cellSetColor,
+  }, {
+    setAdditionalObsSets: setAdditionalCellSets,
+    setObsSetColor: setCellSetColor,
+    setObsColorEncoding: setCellColorEncoding,
+    setObsSetSelection: setCellSetSelection,
   }] = useCoordination(
     COMPONENT_COORDINATION_TYPES[ViewType.FEATURE_VALUE_HISTOGRAM],
     coordinationScopes,
   );
 
   const [width, height, containerRef] = useGridItemSize();
   const [urls, addUrl] = useUrls(loaders, dataset);
+  const [dataOnSelect, setDataOnSelect] = useState([]);
```

Comment:
When calling `onSelect`, the value of `data` is null. It must be connected to the use of debounce in `ExpressionHistogram`. This is why I added the state variable.